### ISSUE TITLE
Feature req: Allow disabling of tablefilter

### ISSUE THREAD

**Issue by atulwy** [webyog/sqlyog-community#1912]
Original [issue 1912](https://code.google.com/p/sqlyog/issues/detail?id=1912) created by webyog on 2013-02-08T08:07:56.000Z:

For tasks which often require access to different databases on the same server, the database filter feature is very handy. But once a db is selected, and I would like to switch to another, the following steps are required to disable the tablefilter in the current db:
1. Scroll to the top of the db list
2. Select the parent node (the db server)
3. Use the db filter

<b>What is the expected output? What do you see instead?</b>
When working in a db, it would be nice to quickly jump to another db. Before the filters were implemented, this could be done by just typing the db name into the db list. But with the new filter system implemented, it filters tables instead.

In summary: An option to always have the db filter accessible, by disabling the tablefilter, or another solution to quickly switch databases.

<b>What version of the product are you using? On what operating system?</b>
SQLyog Ultimate MySQL GUI v10.51
Windows 7 x64
